modules/seqexec/server/src/main/scala/seqexec/server/keywords/DhsClientHttp.scala

```diff
@@ -124,19 +96,72 @@ object DhsClientHttp {
   object DhsError extends ErrorType("DHS_ERROR")
   object InternalServerError extends ErrorType("INTERNAL_SERVER_ERROR")
 
-  implicit def errorTypeDecode: DecodeJson[ErrorType] = DecodeJson[ErrorType]( c =>  c.as[String].map {
+  implicit def errorTypeDecode: Decoder[ErrorType] = Decoder.instance[ErrorType](c =>  c
```

Comment:
Any reason not to use semi derived codecs?

Reply:
Yes. I don't know how.